apps/api/v1/pages/api/invites/_post.ts

```diff
@@ -43,7 +43,6 @@ async function postHandler(req: NextApiRequest, res: NextApiResponse) {
       language: data.language,
       teamId: data.teamId,
       usernameOrEmail: data.usernameOrEmail,
-      isOrg: data.isOrg,
```

Comment:
`isOrg` isn't needed to be passed now. It should be and now is derived by the backend itself.